### ISSUE TITLE
Shadows missing Y value

### ISSUE THREAD

**Issue by alehar9320** [mikaelvesavuori/figmagic#154]
**Describe the bug**
Shadows exported using Figmagic are missing Y value

**To Reproduce**
1. Make a copy of Figmagic's Figma example file
2. Add a square object to shadows frame
3. Add a drop-shadow effect to sqaure object, with value for both X, Y, Blur and Color
4. Make Figmagic fetch from the duplicate of the Figma example file
5. Notice that X, Blur and Color value is retrieved correctly. Y value is not.

**Expected behavior**
All values should be present (X, Y, Blur and Color), as can be found in the local figmagic/figma.json file.

**Screenshots**

figma.json (input from Figma)
![image](https://user-images.githubusercontent.com/5390430/144405147-aa08ff16-2eb1-4c28-926c-3b1b712a85a7.png)

shadows.ts (output)
![image](https://user-images.githubusercontent.com/5390430/144405543-22cc12d5-918a-4054-8fb3-b6aa191d690b.png)

**Desktop:**
 - OS: Windows 10
 - Browser: N/A
 - Version: Figmagic 4.3.5 and 4.4.1 (likely including the range between these two and more)

**Additional context**
TBD

**Comment by mikaelvesavuori** [comment]
This does not seem to entirely true, see

```TypeScript
shadows[NAME] += `${X} ${Y} ${RADIUS} rgba(${R}, ${G}, ${B}, ${A})`;
```

**Comment by mikaelvesavuori** [comment]
This is now fixed in `4.4.2`. Thanks for reporting!

**Comment by kovavka** [comment]
Hi @mikaelvesavuori! I faced the same problem and found [this](https://github.com/mikaelvesavuori/figmagic/blob/main/bin/entities/Token/logic/makeShadowTokens.ts#L62). Could you please fix it?

**Comment by mikaelvesavuori** [comment]
Hey @kovavka! Seems this was a slip-up on my side. I just did a quick adjustment that _should_ address this, released in `4.4.5`.

**Comment by kovavka** [comment]
Thank you so much, it works now. Your project have a best support I've ever seen!
